circle/docker-release-image.sh

```diff
@@ -136,12 +136,15 @@ fi
 # We will only publish debian-9 images to Azure Registry
 if [[ "${DISTRO}" == "debian-9" && -n $AZURE_PROJECT && -n $AZURE_PORTAL_PASS && -n $AZURE_PORTAL_USER ]]; then
```

Comment:
This does not check for `AZURE_PUBLIC_REGISTRY`. Do you think it's safe to assume this won't be set without the other? You could check it here with an `OR`, and skip the loop iteration in the `for` loop below.

Reply:
yes, I think it is safe and they will always be set together.
Also, we are working on pushing the images from a different process, so any change is likely to happen in that other process rather than here, by the time being.